src/test/java/com/sonyericsson/rebuild/matchers/StringParameterValueMatcher.java

```diff
@@ -0,0 +1,30 @@
+package com.sonyericsson.rebuild.matchers;
+
+import hudson.model.StringParameterValue;
+import org.hamcrest.BaseMatcher;
+import org.hamcrest.Description;
+
+public final class StringParameterValueMatcher extends BaseMatcher<StringParameterValue> {
+
+    private final StringParameterValue thisParameterValue;
+
+    StringParameterValueMatcher(StringParameterValue thisParameterValue) {
+        this.thisParameterValue = thisParameterValue;
+    }
+
+    public static StringParameterValueMatcher equalToStringParamValue(String name, String value) {
+        return new StringParameterValueMatcher(new StringParameterValue(name, value));
+    }
+
+    @Override
+    public boolean matches(Object parameterValue) {
```

Comment:
I would say the input should be called `parameter` not `parameterValue` which can be confused with what in fact is `parameter.value`